@here/harp-mapview/lib/Tile.ts

```diff
@@ -1057,6 +1061,38 @@ export class Tile implements CachedResource {
         }
     }
 
+    /**
+     * Gets the tile's bounding box.
+     */
+    get boundingBox(): OrientedBox3 {
+        return this.m_boundingBox;
+    }
+
+    private setBoundingBox(newBoundingBox?: OrientedBox3) {
```

Comment:
Sorry for being picky, but honestly I dislike this kind of setter (especially if we have "real" getter). Worth to note that property setter itself can be also private - like this method. On other side calling:
```
this.setBoundingBox()
```
without arguments seems to be strange... Maybe simply provide another method: `invalidateWorldBox()` that will convert geo to world bounding box, and leave the setter withing convention.

Reply:
The problem is setter and getter must have same access specifier, otherwise you get a ts-lint error. The setter must be however private, code outside the class is not supposed to change the bounding box in the tile. 
I've added this method to keep in sync the bounding box with the worldCenter of the tile, their update must happen in the same single place.

However, I agree that with that name it's weird to call setBoundingBox without arguments. So I'll rename to updateBoundingBox and add a comment clarifying what happens when you don't pass a bounding box.

Reply:
Nice, I like it too!